ui/screens.py

```diff
@@ -1494,6 +1735,10 @@ def setPools(self):
         pools = acct.stakePools
         if len(pools) == 0:
             return
+        # Refresh purchase info
+        pools[0].getPurchaseInfo()
```

Comment:
https://www.dcrstakedinner.com

How long do you think the expiration should be? Because the user could change this via the vsp's web interface, I think too long is not good. Maybe an hour?